api/src/main/java/org/pmiops/workbench/api/WorkspaceController.java

```diff
@@ -100,20 +113,39 @@ private void setCdrVersionId(Workspace workspace,
     }
   }
 
+  private FirecloudWorkspaceId generateFirecloudWorkspaceId(String name) {
```

Comment:
Okay. Do we want to change the router variable names on the front-end once this goes through? And do we want to change the swagger definitions to match it?